.travis.yml

```diff
@@ -57,6 +57,28 @@ jobs:
       language: python
       script: ./.travis.sh includes
       cache: false
+    - name: "Clang-tidy"
```

Comment:
This check seems to take very long on travis. Please move it to the second build stage for efficiency.